.github/resources/scripts/deploy-kfp.sh

```diff
@@ -24,19 +24,27 @@ C_DIR="${BASH_SOURCE%/*}"
 if [[ ! -d "$C_DIR" ]]; then C_DIR="$PWD"; fi
 source "${C_DIR}/helper-functions.sh"
 
+TEST_MANIFESTS=".github/resources/manifests/argo"
 USE_PROXY=false
 
-while getopts ":p-:" OPT; do
-    case $OPT in
-        -) [ "$OPTARG" = "proxy" ] && USE_PROXY=true || { echo "Unknown option --$OPTARG"; exit 1; };;
-        \?) echo "Invalid option: -$OPTARG" >&2; exit 1;;
-    esac
+# Loop over script arguments passed. This uses a single switch-case
+# block with default value in case we want to make alternative deployments
+# in the future.
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+    --deploy-k8s-native)
+      CRD_MANIFESTS="manifests/kustomize/base/crds"
+      TEST_MANIFESTS=".github/resources/manifests/kubernetes-native"
```

Comment:
If --proxy and --deploy-k8s-native are both set then it will not work. Let's perhaps not support the proxy deployment method for Kubernetes native API.